### PR TITLE
Implementation of prime_nu(n) and prime_omega(n) to the ntheory module

### PR DESCRIPTION
**Pull request by gorisaka**
This PR suggests the addition of two Functions to the ntheory module.
- **prime_nu(n)**: calculate the number of distinct prime factors for a positive integer n.
- **prime_omega(n)**: calculate the number of prime factors counting multiplicities for a positive integer n.

Both are counterparts to Wolfram's PrimeNu[n] and PrimeOmega[n]:
https://reference.wolfram.com/language/ref/PrimeNu.html
https://reference.wolfram.com/language/ref/PrimeOmega.html

### CODE REVIEW

File: sympy/ntheory/factor_.py [hunk 1]
```diff
@@ -1842,3 +1842,92 @@ def eval(cls, n, k=1):
...
+
+    If n's prime factorization is:
+
+    .. math ::
+        n = \prod_{i=1}^k p_i^{m_i},
+
+    then ``prime_nu(n)`` or `\\nu(n)` is:
+
+    .. math ::
+        \\nu(n) = k.
```
Comment (asmeurer) [inline sympy/ntheory/factor_.py]: It would be better to make the docstring a raw string (use `r"""` at the beginning).

**Comment by asmeurer** [comment]
Since it's capital Omega should it be `prime_Omega`?

**Comment by asmeurer** [comment]
Also, `primepi` doesn't have an underscore, so maybe we should be consistent with that.

**Comment by asmeurer** [comment]
Regarding the Omega thing, `gamma` is lowercase even though it's an uppercase gamma, so perhaps it's OK.

**Comment by asmeurer** [comment]
You should add some printers for this, particularly Unicode pretty printing and LaTeX (especially since `prime_omega` will print as `prime_ω` by default).

**Comment by asmeurer** [comment]
I have just those comments. Everything else looks good.

**Comment by gorisaka** [comment]
@asmeurer Done. Any comments on the new changes?

**Pull request merged**